app/application-settings/overview.md

```diff
@@ -1,4 +1,2 @@
 The Application settings allows you to save and restore any kind of information related to your application.
-Using application settings methods requires to load "application settings" module.
-
-**API Reference for** [Application Settings](https://docs.nativescript.org/api-reference/modules/_application_settings_.html)
\ No newline at end of file
+Using application settings methods requires to load "application settings" module.
```

Comment:
Let's use consistent capitalisation type across the whole documentation. Previously, we used "Application Settings" so probably this should be changed as well.

Reply:
Regarding the "files with no actual changes". In some of the examples, I added empty end.md files, because of the properly working of the script. Perhaps GitHub has marked them as unneeded change.